main.cpp

```diff
@@ -18,13 +18,35 @@ using namespace clang::ast_matchers;
 using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
+private:
+    Rewriter& castRewriter;
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
+    CastCallBack(Rewriter& rewriter) : castRewriter(rewriter) { }
 
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+        const auto* expression = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");

```

Comment:
What if `expression` is a null pointer? How to protect the program from crashing in such cases?

Reply:
To protect the program from crashing in such cases I need to add this code:
if (expression!= nullptr) {
// some code, for example “return;” 
}